ldms/rules.mk

```diff
@@ -0,0 +1,24 @@
+# 1. Fix duplicate titles when converting to man
+# 	- Remove "NAME" and "=====" underneath
```

Comment:
Why are we doing this dynamically? Why don't we just fix all the rst sources to use the following format?

https://docutils.sourceforge.io/sandbox/manpage-writer/rst2man.txt